roles/ceph-common/tasks/main.yml

```diff
@@ -29,7 +29,7 @@
 
 - name: install ceph
   apt: pkg={{ item }}
-       state=present
+       state=latest
```

Comment:
This was used for ceph upgrade. Now that niraj said we will use pack version instead of stable release version, I think I can specif the package version here instead of using `latest`